omod/src/main/webapp/fragments/field/passwordField.gsp

```diff
@@ -0,0 +1,20 @@
+<%
+    config.require("label")
+    config.require("formFieldName")
+    if (config.classes && config.classes.contains("regex")) {
+        config.require("regex")
+    }
+%>
+
+<p <% if (config.left) { %> class="left" <% } %> >
+    <label for="${ config.id }-field">
+        ${ config.label } <% if (config.classes && config.classes.contains("required")) { %>
+        <span>(${ ui.message("adminui.requiredField.label") })</span> <% } %>
```

Comment:
Move the message code to uicommons module and rename it to *uicommons.requiredField.label* so that one doesn't need to install the adminui module just to get the translation